esbuild.config.mjs

```diff
@@ -15,7 +15,7 @@ esbuild.build({
 	banner: {
 		js: banner,
 	},
-	entryPoints: ['main.ts'],
+	entryPoints: ['main.tsx'],
```

Comment:
recall that we need to `.[jt]sx` for when we use JSX in the plugin for the output